packages/ts-transform/src/utils/ast-node.tsx

```diff
@@ -1,5 +1,12 @@
 import * as ts from 'typescript';
 
+export const createNodeError = (message: string, node: ts.Node) => {
+  throw new Error(`@compiled/css-in-js => ${message}
+
+${node.getText()}
+`);
+};
+
```

Comment:
Moved this method to the top because of a rule that requires functions to be declared before use. Not sure if this is something you want to keep.

Reply:
is ok